docs/getting-started-guides/mesos/index.md

```diff
@@ -217,10 +217,7 @@ Kube-dns is an addon for Kubernetes which adds DNS-based service discovery to th
 The kube-dns addon runs as a pod inside the cluster. The pod consists of three co-located containers:
 
 - a local etcd instance
-- the [skydns][11] DNS server
-- the kube2sky process to glue skydns to the state of the Kubernetes cluster.
-
-The skydns container offers DNS service via port 53 to the cluster. The etcd communication works via local 127.0.0.1 communication
+- the [kube-dns][11] DNS server
```

Comment:
I am not sure about these changes, I assumed with kubedns service there should not be any need to have a glue process. Please correct me if I am wrong, also I could not verify if anything was running on 53 or not. I am looking for answers here.